_rules/html-page-lang-valid-bf051a.md

```diff
@@ -62,15 +62,15 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-This `html` element has a `lang` attribute whose value is a [valid][valid language subtag] [primary language subtag][].
+This `html` element has a `lang` attribute whose value is a [valid][valid language tag] [primary language subtag][].
 
 ```html
 <html lang="fr"></html>
 ```
 
 #### Passed Example 2
 
-This `html` element has a `lang` attribute value that is a [valid language subtag][] even though the [region subtag][] is not.
+This `html` element has a `lang` attribute value that is a [valid language tag][] even though the [region subtag][] is not.
```

Comment:
It is very weird to say that the attribute has a "valid language tag" with an invalid region subtag. It is mixing the definition of "valid" (ours and BCP 47).
"[value] is a valid language tag" => with our definition of valid.
"the region subtag is not [valid]" => with the BCP definition.


This is likely due to the fact that our definition of "valid language tag" is actually meaning "language tag starting with a valid primary subtag".
I agree that "valid language subtag" was not a good name (it is only checking the primary subtag, not any). But I feel that "valid language tag" is not really better.

Reply:
I think it's better; but I agree that the word "valid" isn't ideal. I thought about that when I worked on this, but couldn't come up with anything to improve on it. Either way I think that's separate from what this PR is addressing, can you open an issue for it?

Reply:
I'm not entirely sure why we'll need 2 PRs to fix a single broken name ("valid language tag"), but sure.